src/utils/request/xhr.ts

```diff
@@ -120,7 +132,7 @@ export default function request<T>(
       deregisterCancellationListener = cancelSignal
         .register(function abortRequest(err : CancellationError) {
           clearCancellingProcess();
-          if (!isNullOrUndefined(xhr) && xhr.readyState !== 4) {
+          if (!isNullOrUndefined(xhr) && xhr.readyState !== XMLHttpRequest.DONE) {
```

Comment:
Why is that condition necessary/

Reply:
I have changed `4` to `XMLHttpRequest.DONE` as it is more explicit.
I can't see how xhr can be null or undefined, so it might right to remove it

Reply:
Yes I was talking about the former